CMakeLists.txt

```diff
@@ -88,6 +88,7 @@ option(BUILD_SHARED_LIBS "Build using shared libraries" OFF)
 option(PCAPPP_USE_DPDK "Setup PcapPlusPlus with DPDK. In this case you must also set DPDK_ROOT")
 option(PCAPPP_USE_PF_RING "Setup PcapPlusPlus with PF_RING. In this case you must also set PF_RING_ROOT")
 option(PCAPPP_INSTALL "Install Pcap++" ${PCAPPP_MAIN_PROJECT})
+option(PCAPPP_INSTALL_EXAMPLES "Install Pcap++ examples" OFF)
```

Comment:
Why not just turning the PCAPPP_BUILD_EXAMPLES OFF to not install it?

I'm not against but I don't see a case where we want to build something and not install it after no?

Reply:
Ohk makes sense, but why do you think we need `PCAPPP_INSTALL`?